### DISCUSSION TITLE
How to Send Ephemeral Messages

### DISCUSSION THREAD

**Discussion by LearnTech-Lingo** [discord-jda/JDA#1583] [Questions and Help]
Hi there !

No matter how much I dig through the documentation, I can't figure out how to send an ephemeral message (like Clyde).
If anyone know the command, tell me    : D

(a message like that : )
[https://support.discord.com/hc/article_attachments/1500000616802/mceclip0.png](url)

**Comment by MinnDevelopment** [comment]
You must use [slash commands](https://github.com/DV8FromTheWorld/JDA/pull/1501) in order to send ephemeral messages. Simply put `setEphemeral(true)` on the reply.

**Reply by LearnTech-Lingo** [reply]
can you show me an exemple please .. ?

**Reply by MinnDevelopment** [reply]
There is an example in the linked PR.
